src/main/java/cz/cvut/kbss/termit/util/ConfigParam.java

```diff
@@ -24,6 +24,11 @@
      */
     REPOSITORY_URL("repository.url"),
 
+    /**
+     * URL of the main application repository.
+     */
+    REPOSITORY_PUBLIC_URL("repository.public-url"),
```

Comment:
Other parameters use camelCase. It would be nice to stay consistent when naming the parameters...